addons/html_editor/static/src/fields/html_field.js

```diff
@@ -32,14 +46,66 @@ export class HtmlField extends Component {
                 this.lastValue !== record.data[this.props.name].toString()
             ) {
                 this.state.key++;
+                this.state.containsComplexHTML = this.computeContainsComplexHTML();
             }
         });
+
+        useEffect(
```

Comment:
i prefer just onMounted here

Reply:
no need to remove the load event actually. you can just add the listener with `once`,

Reply:
> i prefer just onMounted here

Why ? It s base the record.data ?

Reply:
> no need to remove the load event actually. you can just add the listener with `once`,



> no need to remove the load event actually. you can just add the listener with `once`,

I m not sure to understand ?